Dockerfile

```diff
@@ -4,9 +4,9 @@ WORKDIR /app
 
 COPY . /app
 
+RUN apt-get update && apt-get install -y libpq-dev build-essential
```

Comment:
This installs the necessary system libraries and development headers for psycopg2 to compile and interact with PostgreSQL. It is not needed if you're using psycopg2-binary. psycopg2-binary comes precompiled and doesn't require these system dependencies.